_includes/manuals/2.2/1.2_release_notes.md

```diff
@@ -32,6 +32,23 @@ Work has been started on creating a new host detail page. The initial implementa
 * Support fetching all records in API ([#29909](https://projects.theforeman.org/issues/29909))
 * graphql: add queries and mutations for settings ([#28920](https://projects.theforeman.org/issues/28920))
 
+#### Hammer CLI
```

Comment:
The sections are ordered alphabetically, can you move this section to between `Foreman` and `Installer` sections?

Reply:
Okay, I thought it should be after API section as per the 2.0 https://github.com/theforeman/theforeman.org/pull/1540 pr .